backend/api/database.py

```diff
@@ -17,3 +17,40 @@ def get_user_by_email(db: Session, email: str):
     if len(user) > 1:
         raise MultipleResultsFound
     return user[0]
+
+
+def get_all_users(db: Session):
+    users = db.query(UserAccount).all()
```

Comment:
Just as a comment as I know when we paired we were trying to recall what was deprecated in sqlalchemy 1 vs 2. Per [these docs](https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-orm-usage), it looks to me like while it is the v1 style it will continue being supported.